### ISSUE TITLE
Hydrogen got NODE_MODULE_VERSION mismatch after updating atom to 1.19.0

### ISSUE THREAD

**Issue by gepcel** [nteract/hydrogen#937]
Yesterday, my Atom got updated to 1.19.0 automatically. And hydrogen stopped working. So I reinstalled hydrogen (by uninstall and then install again is Settings-Packages). Then after launching atom, I got this error (from `Incompatible Packages` atom shows). What should I do?

--------
Hydrogen 1.19.0

zeromq@4.5.0 – The module '\\?\C:\Users\way\.atom\packages\Hydrogen\node_modules\zeromq\build\Release\zmq.node' was compiled against a different Node.js version using NODE_MODULE_VERSION 49. This version of Node.js requires NODE_MODULE_VERSION 53. Please try re-compiling or re-installing the module (for instance, using `npm rebuild` or`npm install`).

--------

Versions (from `atom --version` and `apm --version`:
Windows 10 x64
Atom 1.19.0
Electron: 1.6.9
Chrome: 56.0.2924.87
Node: 7.4.0
Hydrogen 1.19.0
apm: 1.18.2
npm 3.10.10
python 3.6.1
git 2.11.1.windows.1

**Comment by BenRussert** [comment]
Is there a button on the incompatible package message that says something like "rebuild modules"?

**Comment by gepcel** [comment]
Wow, first of all, it worked.

 I've reinstalled these (there is another package incompatible) packages multiple times. And googled around for a very long time. And searched the atom git repo. 

I've noticed there are something there but didn't pay attention what it says, because of the color of the text. I even tried to click the last two buttons, just without trying the first button for even once. Thank you very much.

![image](https://user-images.githubusercontent.com/4179106/29150007-502e7208-7daa-11e7-8693-b3f6514f907d.png)
